### ISSUE TITLE
Won't save after geo locating

### ISSUE THREAD

**Issue by paul-frost** [ethercreative/simplemap#47]
I enter a postcode in the search bar, select the result from the popup, then when I click Save nothing happens. But if I select "save and continue editing" it saves.

I've tried this a few times and sometimes if I put in a new postcode it works first time. But if I then put in the original postcode it doesn't.

There are no errors or warnings in the console.

**Comment by paul-frost** [comment]
I've updated SimpleMap to 1.6.2 from 1.6.1 and craft to 2.6.2987 from 2.6.2986 and it now seems to be working.
Firefox: 54.0.1 & Chrome: 59.0.3071.115
OS: macOS 10.12.5
There are a pile of plugins: Cache Buster, CP Sortable Custom Columns, Feed Me. Field Manager, Help & Tips, Image Resizer, Import, K4 Encode Emails, Placid, Preparse Field and Redactor Word Count.
I did have SquareBit Maps installed but disabled, as this was the plugin I was using before I switched to SimpleMaps. I had since uninstalled it.
